api/firmware/device.go

```diff
@@ -332,11 +332,17 @@ func (device *Device) Product() common.Product {
 }
 
 // SupportsETH returns true if ETH is supported by the device api.
-// coinCode is eth/teth/reth or eth-erc20-xyz, ...
+// coinCode is eth/teth/reth/goeth or eth-erc20-xyz, ...
 func (device *Device) SupportsETH(chainID uint64) bool {
 	if *device.product != common.ProductBitBox02Multi {
 		return false
 	}
+	if device.version.AtLeast(semver.NewSemVer(9, 10, 0)) {
+		switch chainID {
+		case 1, 3, 4, 5: // mainnet, ropsten, rinkeby, goerli
+			return true
+		}
```

Comment:
you can just return true here for all chainIDs, since 9.10.0 accepts all of them.

Reply:
Cool, done!